ui/components/multichain/pages/gator-permissions/components/review-gator-permission-item.tsx

```diff
@@ -73,6 +73,11 @@ type ReviewGatorPermissionItemProps = {
    * The function to call when the revoke is clicked
    */
   onRevokeClick: () => void;
+
+  /**
+   * Whether this permission has a pending revoke click (temporary UI state)
+   */
+  isPendingRevokeClick?: boolean;
```

Comment:
nit: I think this name is a little confusing - maybe `hasRevokeBeenClicked` would be more clear.